[2024-12-16T00:00:08.240+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-16T00:00:08.252+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-16T00:00:08.253+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-16T00:00:08.270+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_data> on 2024-12-15 00:00:00+00:00
[2024-12-16T00:00:08.276+0000] {standard_task_runner.py:57} INFO - Started process 500 to run task
[2024-12-16T00:00:08.280+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'store_data', 'scheduled__2024-12-15T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmprlal5ypu']
[2024-12-16T00:00:08.283+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask store_data
[2024-12-16T00:00:08.366+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.store_data scheduled__2024-12-15T00:00:00+00:00 [running]> on host 6cd16bdff3c4
[2024-12-16T00:00:08.508+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='store_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T00:00:00+00:00'
[2024-12-16T00:00:08.561+0000] {data_warehouse.py:50} INFO - Database tables initialized successfully
[2024-12-16T00:00:08.562+0000] {data_warehouse.py:66} INFO - Saving data to warehouse table: covid_daily_stats
[2024-12-16T00:00:08.603+0000] {data_warehouse.py:92} INFO - Successfully saved 231 records to covid_daily_stats
[2024-12-16T00:00:08.604+0000] {python.py:194} INFO - Done. Returned value was: Data stored successfully
[2024-12-16T00:00:08.641+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=store_data, execution_date=20241215T000000, start_date=20241216T000008, end_date=20241216T000008
[2024-12-16T00:00:08.667+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T00:00:08.695+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-16T20:18:25.850+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-16T20:18:25.862+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2024-12-16T20:18:25.863+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-16T20:18:25.879+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_data> on 2024-12-15 00:00:00+00:00
[2024-12-16T20:18:25.884+0000] {standard_task_runner.py:57} INFO - Started process 228 to run task
[2024-12-16T20:18:25.889+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'store_data', 'scheduled__2024-12-15T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmpm0y2xeut']
[2024-12-16T20:18:25.893+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask store_data
[2024-12-16T20:18:25.973+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.store_data scheduled__2024-12-15T00:00:00+00:00 [running]> on host 6c524df05549
[2024-12-16T20:18:26.114+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='store_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T00:00:00+00:00'
[2024-12-16T20:18:26.167+0000] {data_warehouse.py:50} INFO - Database tables initialized successfully
[2024-12-16T20:18:26.168+0000] {data_warehouse.py:66} INFO - Saving data to warehouse table: covid_daily_stats
[2024-12-16T20:18:26.217+0000] {data_warehouse.py:92} INFO - Successfully saved 231 records to covid_daily_stats
[2024-12-16T20:18:26.218+0000] {python.py:194} INFO - Done. Returned value was: Data stored successfully
[2024-12-16T20:18:26.267+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=store_data, execution_date=20241215T000000, start_date=20241216T201825, end_date=20241216T201826
[2024-12-16T20:18:26.314+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T20:18:26.343+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
